app/components/file-list.js

```diff
@@ -4,52 +4,41 @@ import React from 'react'
 import styled from 'styled-components'
 import bytes from 'prettier-bytes'
 
-const FileListContainer = styled.div`
-  :host {
-    min-height: 5rem;
-  }
-`
-
 const FileListTable = styled.table`
-  :host {
-    td {
-      padding: 0.25rem 0.5rem;
-    }
-    tr:odd td {
-      background-color: var(--color-neutral-04);
-    }
+  width: 100%;
+  border-collapse: collapse;
+  tr:odd td {
+    background-color: var(--color-neutral-04);
```

Comment:
Shouldn't this make every other line different? In my test, each line has the same color?!

<img width="762" alt="screen shot 2018-11-30 at 14 03 24" src="https://user-images.githubusercontent.com/914122/49269514-c0bb8d00-f4a8-11e8-8503-47ec1ead3068.png">

Reply:
this styling is not working before my PR. it is other issue.

Reply:
However, my PR was not conscious for that style.
I will work to the this issue.

Reply:
@martinheidegger san
I FIXED. PLEASE RE-REVIEW.